api/base/serializers.py

```diff
@@ -541,11 +545,13 @@ def resolve(self, resource):
         """
         Resolves the view for target node or target comment when embedding.
         """
-        view_info = self.view_map.get(resource.target._name, None)
+        view_info = self.view_map.get(resource.target.referent._name, None)
```

Comment:
Right. I guess my question could be better phrased as "Will `TargetField` ever be used for anything other than comments?"